lib/node_modules/@stdlib/number/float64/base/exponent/test/test.native.js

```diff
@@ -0,0 +1,113 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var BIAS = require( '@stdlib/constants/float64/exponent-bias' );
+var randu = require( '@stdlib/random/base/randu' );
+var round = require( '@stdlib/math/base/special/round' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var bits = require( '@stdlib/number/float64/base/to-binary-string' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+
+
+// VARIABLES //
+
+var exponent = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( exponent instanceof Error )
+};
+
+
+// TESTS //
+
+tape( 'main export is a function', opts, function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof exponent, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns an integer', opts, function test( t ) {
+	var w = exponent( 3.14 );
+
+	t.equal( typeof w, 'number', 'returns a number' );
+	t.equal( w % 1, 0, 'returns an integer' );
+
+	t.end();
+});
```

Comment:
Testing for an integer is more precise; however, my preference would be for the tests in this file and test.js to match.

If you want, you can update `test.js` accordingly. Or you can update this file to match `test.js`. If testing for an integer, use `@stdlib/assert/is-integer' ).isPrimitive`.

Reply:
Sure, I will do that.